railgun/manager/run.go

```diff
@@ -86,6 +86,7 @@ func Run(ctx context.Context, c *config.Config) error {
 		HealthProbeBindAddress: c.ProbeAddr,
 		LeaderElection:         c.EnableLeaderElection,
 		LeaderElectionID:       c.LeaderElectionID,
+		SyncPeriod:             &c.SyncPeriod,
```

Comment:
Can you please also add the code applying this parameter ?

Reply:
I'm sorry but I don't follow what is missing, can you help me? Perhaps provide a file name or something that could send me in the right direction?

Reply:
You need code function/module that use this "SyncPeriod" parameter.

Reply:
This is it. This is where it's being used. As `ctrl.Options` input, and used below on line 104 by `ctrl.NewManager(kubeconfig, controllerOpts)`.

Reply:
SyncPeriod is a new option within controllerOpts, which code is using controllerOpts.SyncPeriod ? write that code or you want modify some existing code

Reply:
> which code is using controllerOpts.SyncPeriod?

This code is:

https://github.com/kubernetes-sigs/controller-runtime/blob/1e4d87c9f9e15e4a58bb81909dd787f30ede7693/pkg/manager/manager.go#L316

Reply:
oh, ha. Sorry, my bad, I was not aware we have been using something undefined.
mark this as bug for you.